answer/damegane/JS/10.3sec/10.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>3sec</title>
+  <style>
+    .container {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <div class="container">
+    <input type="button" value="押すなよ" onclick="thridsec()">
+  </div>
+  <script>
+    const thridsec = () => {
+      sleep(3000);
+      alert('3秒経ちました');
+    };
+
+    const sleep = (time) => {
```

Comment:
nits:
知っているかもしれませんし変更する必要も無いですがアロー関数で引数が1つの時は
```suggestion
    const sleep = time => {
```
のように()を省略することもできますよ
一応:eyes: